projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra.stats.draw_strength.tooltip_extended": "伤害计算基于原版的弹射物行为。伤害的下限取决于模块与升级，上限是：§f§o上限 = 下限 * 1.5 + 1§r§7。由不完全拉开的弓射出的弹射物伤害基于下限计算，例如：从 50%% 拉开的弓射出的弹射物会造成 50%% 的伤害下限。",
+    "tetra.stats.ability_damage": "伤害",
+    "tetra.stats.ability_damage.tooltip": "这个物品的特殊功能有 §e%s 点的基础伤害",
+    "tetra.stats.bleeding": "流血",
+    "tetra.stats.bleeding.tooltip": "攻击生物时有 30%% 的概率使它们以 §e%s§r 的伤害持续流血 2 秒",
+    "tetra.stats.backstab": "背刺",
+    "tetra.stats.backstab.tooltip": "从背后击中目标会造成致命一击，造成 §e%s§r 点额外伤害",
+    "tetra.stats.armor": "护甲",
+    "tetra.stats.armor.tooltip": "持有时提供 §e%s§r 点护甲值",
+    "tetra.stats.toughness": "坚韧",
+    "tetra.stats.toughness.tooltip": "在持有时提供 §e%s§r 点坚韧值",
+    "tetra.stats.blocking": "防御",
+    "tetra.stats.blocking.tooltip": "按住右键可以防御受到的攻击和弹射物",
+    "tetra.stats.blocking_duration.tooltip": "按住右键可以防御受到的攻击和弹射物，最多持续 §e%s§r 秒",
+    "tetra.stats.blocking.tooltip_extended": "如果这个物品有第二个特殊功能，那么那个功能可以通过按下 shift + 右键，或在右键松开时按住 shift（这会触发一个延长了 50%% 的冷却）。",
+    "tetra.stats.blocking_reflect": "反伤",
+    "tetra.stats.blocking_reflect.tooltip": "§e§7防御近战伤害时有 §e%s§r 的概率对攻击者造成 §e%s§r 的伤害，并产生击退效果。",
+    "tetra.stats.bashing": "猛击",
+    "tetra.stats.bashing.tooltip": "按右键可以猛击生物。猛击会击退生物，击晕它并造成 §e100%%§r 的伤害\n \n§7击退倍率：§e%s§r\n§7击晕持续时间：§e%s§7 秒",
+    "tetra.stats.throwable": "可投掷",
+    "tetra.stats.throwable.tooltip": "可以按住右键来投掷。击中实体时造成 §e%s%%§r 的伤害。击中方块时，如果物品的采集等级足够高并且挖掘速度大于方块硬度，击中方块时会破坏此方块。",
+    "tetra.stats.jab": "猛刺",
+    "tetra.stats.jab.tooltip": "副手持有时可以右键攻击生物，造成 §e%s§r 点伤害并产生击中效果。触发等于物品攻击力的冷却时间。",
+    "tetra.stats.armorPenetration": "穿甲",
+    "tetra.stats.armorPenetration.tooltip": "击中目标将造成至少 §e%s§r 点伤害，无视护甲",
+    "tetra.stats.unarmoredDamage": "切割",
+    "tetra.stats.unarmoredDamage.tooltip": "攻击无护甲的生物时造成 §e%s§r 的额外伤害",
+    "tetra.stats.knockback": "击退",
+    "tetra.stats.knockback.tooltip": "增加 §e%s§r 倍的击退距离",
+    "tetra.stats.looting": "抢夺",
+    "tetra.stats.looting.tooltip": "增加杀死生物的掉落物",
+    "tetra.stats.fiery": "炽热",
+    "tetra.stats.fiery.tooltip": "攻击生物时使它着火 §e%s§r 秒",
+    "tetra.stats.smite": "亡灵杀手",
+    "tetra.stats.smite.tooltip": "在攻击亡灵生物时造成 §e%s§r 的额外伤害",
+    "tetra.stats.arthropod": "节肢杀手",
+    "tetra.stats.arthropod.tooltip": "在攻击节肢生物时造成 §e%s§r 的额外伤害并附加 1-§e%s§r 秒的 缓慢 IV 效果",
+    "tetra.stats.sweeping": "横扫之刃",
+    "tetra.stats.sweeping.tooltip": "攻击生物时也对所有附近的敌人造成 §e%s§r 的武器伤害",
+    "tetra.stats.unbreaking": "耐久",
+    "tetra.stats.unbreaking.tooltip": "使用时有几率不减少耐久\n \n§7耐久等级：§e%s§7\n耐久不减少概率：§e%s%%",
+    "tetra.stats.mending": "经验修补",
+    "tetra.stats.mending.tooltip": "消耗获得的经验球以修复耐久，每点经验修复 §e%s§r 耐久\n \n§7只在左手或右手持有时有效",
+    "tetra.stats.silkTouch": "精准采集",
+    "tetra.stats.silkTouch.tooltip": "破坏时掉落方块本身，而不是一般的掉落物",
+    "tetra.stats.fortune": "时运",
+    "tetra.stats.fortune.tooltip": "在采集或回收方块时增加掉落几率",
+    "tetra.stats.flame": "火矢",
+    "tetra.stats.flame.tooltip": "击中实体时使它着火 §e%s§r 秒，弹射物会点燃tnt和营火一类的方块",
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
    "tetra.stats.flame.tooltip": "击中实体时使它着火 §e%s§r 秒，弹射物会点燃 TNT 与营火一类的方块",
```